uktrade/data-hub-frontend
# Reactify OMIS quote page

## Description of change

The OMIS quote page has been reactified. The additions diff is so large because a new dependency was added.

## Test instructions

Go to the quote page for a draft order. You should be able to submit the quote.
Go to the quote page for an order where the quote is awaiting acceptance. You should be able to withdraw the quote.
Go to the quote page for any other non-cancelled order. The quote and T&C's should render as usual.

## Screenshots

This is an old quote which is why DIT is mentioned in the content.

![Screenshot 2024-02-12 at 12-23-50 Quote - RVK689_22 - Orders (OMIS) - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/e854d4fd-2148-4cde-95b4-b140efd7d5a1)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
